application/schemas/templates/components.py

```diff
@@ -28,8 +37,41 @@ class Component(BaseModel):
                     'a higher priority.',
     )
 
```

Comment:
You forgot to add:
```
    class Config:
        extra = Extra.forbid
```
This setting forbids unknown parameters in mode. By default in is silently ignored.